src/test/QuantumGravityBridge.t.sol

```diff
@@ -35,7 +35,8 @@ contract RelayerTest is DSTest {
 
         validators.push(Validator(cheats.addr(testPriv1), votingPower));
         bytes32 hash = computeValidatorSetHash(validators);
-        bridge = new QuantumGravityBridge(initialVelsetNonce, (2 * votingPower) / 3, hash);
+        bridge = new QuantumGravityBridge();
+        bridge.initialize(initialVelsetNonce, (2 * votingPower) / 3, hash);
```

Comment:
unrelated to this change question

I don't think so, but is there ever a scenario where we have to be worried about this math?

Reply:
This is just a test, and we're defining ourselves the voting power for that validator. So it's okey